.storybook/preview.js

```diff
@@ -0,0 +1,9 @@
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+}
```

Comment:
(this is all default -- no changes made to this file)

Reply:
did the linter run on this generated file? i see a missing newline-at-end

Reply:
no, i haven't set up linting yet, but it's up next! i wanted to have something to lint first 😄